src/prefect/core/task.py

```diff
@@ -180,8 +182,8 @@ def __init__(
         result_handler: "ResultHandler" = None,
         state_handlers: List[Callable] = None,
         on_failure: Callable = None,
+        log_stdout: bool = True,
```

Comment:
As @wagoodman pointed out in the issue, it might be better to use `False` as a default here